eeg_cofitage/dataset_description.json

```diff
@@ -0,0 +1,28 @@
+{
+  "Name": "COFITAGE BIDS example",
+  "BIDSVersion":"1.1.2-bep006",
```

Comment:
Can you update this to reflect BIDS 1.2? --> BEP006 has been merged into BIDS

Reply:
Will do

Reply:
Done